### DISCUSSION TITLE
Copilot not working anymore

### DISCUSSION THREAD

**Discussion by JPG-Digital** [community/community#120725] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi Everyone,

My Copilot stopped working two weeks ago and there is nothing I can find on the subject.

I have a paid and valid subscription.
Copilot logs in when opening a project in Visual Studio 2022 17.9.6
 
![image](https://github.com/community/community/assets/123067128/b2bad927-747b-4c16-b1f9-8676a5a3d6c7)

Output also suggests successful connection

![image](https://github.com/community/community/assets/123067128/5573bbf7-367a-4468-8236-a52b071da38e)

But when Copilot needs to connect and fire it comes back with the following. 

![image](https://github.com/community/community/assets/123067128/e06c2633-bd11-44ab-b51d-eeb9c0056891)

See the engine -->> https://copilot-proxy.githubusercontent.com/v1/engines/copilot.codex

I see this was deprecated by Github here. 
https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-network-errors-for-github-copilot

![image](https://github.com/community/community/assets/123067128/b7709a75-5518-4116-b175-8a064e589ef3)

I am using the latest CoPilot Extension I can find.

How can I change the endpoint? 
Is that the issue or do I have to look somewhere else?

**Comment by Matt-PMCT** [comment]
I am experiencing the same issue in PHPStorm, constant disconnections and "first certificate" errors.

**Reply by lada-olc** [reply]
Same in PyCharm :-(

**Reply by Matt-PMCT** [reply, marked as answer]
I found a solution here related to the ESET Antivirus if you have the ability to change settings. (https://github.com/orgs/community/discussions/120205#discussioncomment-9165225) It looks like maybe Co-Pilot started checking certificate names for some reason and ESET monitoring inserts their own.

**Reply by lada-olc** [reply]
Thank's a lot, I added the ignore rule and it works!

**Reply by Matt-PMCT** [reply]
It still is going to be really annoying because that plug-in is stored in a relatively temporary location. Every time JetBrains releases an update, we will have to go add a new ignore rule... Hopefully, a more permanent solution is introduced...

**Reply by JPG-Digital** [reply]
Thank you for the reply and sharing.
All is working again.
